nio/responses.py

```diff
@@ -789,6 +795,28 @@ def create_error(parsed_dict):
         return UpdateDeviceError.from_dict(parsed_dict)
 
 
+@attr.s
+class ProfileGetDisplayNameResponse(Response):
+    displayname = attr.ib(type=Optional[str], default=None)
```

Comment:
Adding a class docstring here would be nice as well. I'll merge this without one but I prefer it to be there.